cms/templates/js/highlights-editor.underscore

```diff
@@ -1,5 +1,32 @@
 <form>
 <h3 class="modal-section-title" id="highlights_label"><%- gettext('Section Highlights') %></h3>
+    <div class="highlights-info">
+
+<% if (highlights_preview_only) { %>
+    <p><b>
+    <%- gettext('We are testing this feature right now. So messages will not actually be sent at this time.') %>
```

Comment:
Trying this again:

This feature is currently in testing. Course teams can enter highlights, but learners will not receive email messages.